### ISSUE TITLE
BREAD Error : Controller does not exist 

### ISSUE THREAD

**Issue by mjsorribas** [thedevdojo/voyager#1216]
- Laravel Version: 5.3.30
- Voyager Version: 0.11.10
- PHP Version: 5.6.30
- Database Driver & Version:
Mysql 5.6
-OS: Mac Os X Yosemite.

### Description:
when i create a new table, the table is created fine.The migration file created fine too. But the model file is empty and never voyager create controller file o views files.
Model File:
```
<?php
namespace App;
use Illuminate\Database\Eloquent\Model;
class UsuariosPago extends Model
{
}
```
### Steps To Reproduce:
install steps  and use for first time.

**Comment by fletch3555** [comment]
What are you expecting to find in the Model file? That one will work perfectly fine as-is.

**Comment by mjsorribas** [comment]
Ok, and why not working?
this error appears when i try to access to the crud:
```
Sorry, the page you are looking for could not be found.

1/1
NotFoundHttpException in RouteCollection.php line 161:
in RouteCollection.php line 161
at RouteCollection->match(object(Request)) in Router.php line 766
at Router->findRoute(object(Request)) in Router.php line 621
at Router->dispatchToRoute(object(Request)) in Router.php line 607
at Router->dispatch(object(Request)) in Kernel.php line 268
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 104
at Pipeline->then(object(Closure)) in Kernel.php line 150
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 117
at Kernel->handle(object(Request)) in index.php line 54
at require_once('/Users/mjsorribas/Documents/Proyectos/demo/public/index.php') in server.php line 21
```

**Comment by fletch3555** [comment]
That's a routing error.  What URL are you trying to go to? and what's the output of `php artisan route:list`?

**Comment by mjsorribas** [comment]
The table is "usuarios_pagos" . The url slug is "usuarios-pagos" . in the builder menu i put "/admin/usuarios-pagos".
The model name is "App\UsuariosPago" and the controller is UsuariosPagoController" .
The controller file UsuariosPagoController don't exist.
The error at run php artisan route:list   is this:
```
#php artisan route:list                                                       
  [ReflectionException]                                             
  Class App\Http\Controllers\UsuariosPagoController does not exist 
```

**Comment by ticouto5290** [comment]
having same error...
any advise ?

**Comment by matt-shirey** [comment]
All you have to do is run the ```php artisan make:controller <controller_name>``` and your controller will be ready.  When you use the database interface inside Voyager, it doesn't create your controller for you.  You have to do that.

**Comment by mjsorribas** [comment]
It's really necesary the automatic creation of this controller. It's a problem when you don't have access to the console for example.

**Comment by fletch3555** [comment]
The controller will not be automatically created.  Voyager will either use the controller you enter into the BREAD configuration (but you're still responsible for creating it), or it will use the default `VoyagerBreadController`.  I'm going to close this for now due to age and inactivity.  If this is still an issue after the 1.0 release, please create a new issue

**Comment by cedlinx** [comment]
Although this thread is closed, I think I should share my solution nonetheless as this shows up at the top of a Google search for this error.

I am new to Laravel and I just installed Voyager after creating my models, controllers and relationships. And then when I created BREAD in Voyager for one of my Models, it was successful. However, when I clicked BROWSE in Voyager to browse the Model, I got an error saying the Controller does not exist.

**Here's what I did while creating BREAD**
I entered AreaController in the Controller Name text box. And this is what returned the error (NOTE: replace AreaController with your controller name)

**The Fix**
I went back to EDIT BREAD for my Area Model and included the full path in the Controller Name text box as follows:

**App\Http\Controllers\AreaController**

And voila! problem solved.
But then, I got a blank page (without any error messages) which I think is because I have not created any views. I may be wrong but I posted this as soon as I got past that error (since I was still on this page). So I would now proceed with my coding in the hope that that's it.

I hope this helps someone.
COA.
